src/app/component/user/components/profile/edit-profile/edit-profile.component.html

```diff
@@ -1,79 +1,90 @@
-            </div>
-            <div class="input-block">
-                <label for="credo">{{"user.edit-profile.title" | translate }}</label>
-                <textarea placeholder="{{userInfo.userCredo}}"
-                          formControlName = "credo"
-                          id="credo"></textarea>
-                <span>{{"user.edit-profile.title-warning" | translate }}</span>
-            </div>
-        </div>
-        <div class="wrapper">
-            <p>{{"user.edit-profile.links" | translate }}</p>
-            <app-social-networks [socialNetworks]="socialNetworks" (socialNetworksChange)="emitSocialLinks($event)"></app-social-networks>
-        </div>
-        <div class="privacy-wrapper">
-            <p class="title">{{"user.edit-profile.privacy" | translate }}</p>
-            <ul>
-                <li>
-                    <label class="label-wrapper">
-                        <input type="checkbox"
-                             formControlName="showLocation">
-                        <span class="checkmark"></span>
-                        {{"user.edit-profile.location" | translate }}
-                    </label>
-                </li>
-                <li>
-                    <label class="label-wrapper">
-                        <input type="checkbox"
-                             formControlName="showEcoPlace">
-                        <span class="checkmark"></span>
-                        {{"user.edit-profile.eco-place" | translate }}
-                    </label>
-                </li>
-                <li>
-                    <label class="label-wrapper">
-                        <input type="checkbox"
-                             formControlName="showShoppingList">
-                        <span class="checkmark"></span>
-                        {{"user.edit-profile.shoping-list" | translate }}
-                    </label>
-                </li>
-            </ul>
-        </div>
-        <div class="buttons">
-            <button type="button" (click)="cancel()">{{"user.edit-profile.btn.cancel" | translate }}</button>
-            <button type="submit" [disabled]="!editProfileForm.valid">{{"user.edit-profile.btn.save" | translate }}</button>
-        </div>
-    </form>
+  <h2>{{ "user.edit-profile.edit-profile" | translate }}</h2>
+  <app-personal-photo></app-personal-photo>
+  <form
+    #formEditProf
+    [formGroup]="editProfileForm"
+    (ngSubmit)="onSubmit()"
+    (keydown.enter)="$event.preventDefault()"
+  >
+    <div class="wrapper">
+      <p>{{ "user.edit-profile.info" | translate }}</p>
+      <div class="input-block" [ngStyle]="{ 'margin-bottom': '41px' }">
+        <label for="name">{{ "user.edit-profile.name" | translate }}</label>
+        <input
+          type="text"
+          formControlName="name"
+          placeholder="{{ userInfo.name.first + ' ' + userInfo.name.last }}"
+          pattern="^(?!\s|.*\s$).*$"
```

Comment:
This line has been added

Reply:
so, just to be sure, now the user will not be able to enter more than one space ? right?

Reply:
Exactly. This feature has been added with that commit 146f945cfd68c1d60d5070cd1e4c3abe545f75aa